config.py

```diff
@@ -475,6 +475,33 @@
             '.*[0-9]+.*',
         ],
     ),
+    
+    fi = dict(
+        lang_name = 'Suomi',
+        lang_dir = 'ltr',
+        database = 'fiwiki_p',
+        wikipedia_domain = 'fi.wikipedia.org',
+
+        beginners_link = 'https://fi.wikipedia.org/wiki/Wikipedia:Merkitse_l%C3%A4hteet',
+        beginners_link_title = 'Wikipedia:Merkitse lähteet',
+        lead_section_policy_link = '',
+        lead_section_policy_link_title = '',
+
+        citation_needed_category = 'Lähteettömät_artikkelit',
+        citation_needed_templates = EN_CITATION_NEEDED_TEMPLATES + [
+            'Lähde',
+            'Lähde tarkemmin',
+            'Kenen mukaan',
+        ],
+        citation_needed_template_name = 'lähde',
```

Comment:
Small detail, but can you please add the question mark at the end? This string is how the templates above end up rendered in Citation Hunt, and I try to have it look the same as in Wikipedia, which uses `lähde?` as far as I can tell.